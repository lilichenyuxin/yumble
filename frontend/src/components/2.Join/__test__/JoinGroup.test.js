import React from 'react';
import renderer from 'react-test-renderer';
import JoinPage from '../JoinGroup';
import {BrowserRouter} from 'react-router-dom';

test('Join Page snapshot testing', () => {
  const component = renderer.create(
      <BrowserRouter>
        <JoinPage />
      </BrowserRouter>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
