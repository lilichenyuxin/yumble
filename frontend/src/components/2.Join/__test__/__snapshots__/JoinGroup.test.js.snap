// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Join Page snapshot testing 1`] = `
Array [
  <h1
    className="Title"
  >
    yumble
  </h1>,
  <div
    className="MakeCentre"
  >
    <div
      className="GroupCode_Box"
    >
      <div
        className="AddMargin"
      >
        Enter group code
      </div>
      <div>
        <input
          className="JoinGroup"
          fontSize={3}
          onChange={[Function]}
          placeholder=" P6aPE"
          type="text"
        />
      </div>
      <div
        className="Username_Box"
      >
        <div
          className="AddMargin"
        >
            Enter Name
        </div>
        <div>
          <input
            className="JoinGroup"
            fontSize={3}
            onChange={[Function]}
            placeholder=" Alex"
            type="text"
          />
        </div>
      </div>
      <button
        className="GoButton"
        onClick={[Function]}
      >
        Go
      </button>
    </div>
  </div>,
  <a
    href="/"
    onClick={[Function]}
  >
    <button
      className="SmallBtn"
      id="BackButton"
    >
      Back
    </button>
  </a>,
  <button
    className="SmallBtn"
    id="HelpButton"
    onClick={[Function]}
  >
    help?
  </button>,
  "",
]
`;
